torch/_refs/fft.py

```diff
@@ -130,6 +134,7 @@ def _fft_c2r(
 
 def _fft_r2c(
     func_name: str,
+    out_defined: bool,
```

Comment:
In core, these accept the `out` tensor as a param, but here I just pass a `bool` to avoid passing the tensor around (which is the point of having out_wrapper).